packages/deployment/test/fixtures/aliases.json

```diff
@@ -0,0 +1,47 @@
+{
+  "Lambdas": {
+    "VersionUpTest": {"hash": "CurrentVersionHash"},
+    "HelloWorld": {"hash": "OnlyVersionhash"}
+  },
+  "Aliases":[
+    [
+      {
```

Comment:
take it or leave it comment.  I reodered the V13 & V12 objects because I think you don't know what order you receive them in.  I know you sort so it still passes, but it might be more obvious or obscure depending on who you are.

Reply:
Agreed, that might make it more explicit to someone perusing the tests, and a more valid fixture.